pkg/kubectl/cmd/util/clientcache.go

```diff
@@ -53,20 +53,18 @@ type ClientCache struct {
 
 	matchVersion bool
 
-	defaultConfigLock sync.Mutex
-	defaultConfig     *restclient.Config
+	mutex         sync.Mutex
```

Comment:
@smarterclayton what do you think of this? or we should still use the old name `defaultConfigLock`?